tests/modules/mlstudio/cdk/test_sagemaker_studio_stack.py

```diff
@@ -21,6 +24,7 @@ def __init__(self, scope, id, env, **kwargs):
             **kwargs,
         )
         self._environment = env
+        self._db = db
```

Comment:
This is needed for this right?
<img width="977" alt="image" src="https://github.com/awslabs/aws-dataall/assets/71252798/ed728485-7b2b-4992-82b9-bab80c5d12bb">

Reply:
Yep!